src/main/resources/beam-template.conf

```diff
@@ -631,6 +631,8 @@ beam.physsim.network.maxSpeedInference {
   type = "String | MEAN"
 }
 
+beam.physsim.network.removeIslands = "boolean | true"
```

Comment:
Let's add some explanation for this new config key, just in a rows above in comment.